packages/libxml2.rb

```diff
@@ -2,39 +2,27 @@
 
 class Libxml2 < CMake
   description 'Libxml2 is the XML C parser and toolkit developed for the Gnome project.'
-  homepage 'http://xmlsoft.org/'
-  version '2.12.7-icu74.2'
+  homepage 'https://gitlab.gnome.org/GNOME/libxml2'
+  version '2.13.2'
   license 'MIT'
   compatibility 'all'
   source_url 'https://gitlab.gnome.org/GNOME/libxml2.git'
-  git_hashtag "v#{version.split('-').first}"
+  git_hashtag "v#{version}"
   binary_compression 'tar.zst'
 
   binary_sha256({
-    aarch64: '80c10f807960e0efb2a1f8d026eab4a9bfdfffc9001aa432c1443776f677987e',
-     armv7l: '80c10f807960e0efb2a1f8d026eab4a9bfdfffc9001aa432c1443776f677987e',
-       i686: '6c0135a19ae7228de19058d5e94dbad7438e80c213cb036eeec294939eccfafe',
-     x86_64: 'b32144d78d066b4031b8ebe1b84b8608f43ef72e9da0c3215fd09e101ee12a20'
+    aarch64: 'eb69071220e11d4f0fd61895b6c46e4be71ec34a0c8a33ff991180437c83f2b9',
+     armv7l: 'eb69071220e11d4f0fd61895b6c46e4be71ec34a0c8a33ff991180437c83f2b9',
+       i686: '199c9ec45327f3be3f402f08023fea2cd9dc1f04ca65e5eac0a2018a26dfd41b',
+     x86_64: '91e68d3cf4eb408735aca931cbe23d7c52dc5f9d3e9084255910d203443507fe'
   })
 
   depends_on 'gcc_lib' # R
   depends_on 'glibc' # R
-  depends_on 'icu4c' # R
+  depends_on 'icu4c'
```

Comment:
Not a runtime dep?

Reply:
I thought the '# R' represented that the dependency was set using getrealdeps.rb-- given that we can confirm this is a dependency without doing that, I removed the comment.

Reply:
`# R` was for being added for runtime deps, which the getrealdeps.rb was adding automatically.

It doesn't catch all of them though, and it does help distinguish deps added before we started using that script... I would keep the `# R` on deps that are runtime deps... Otherwise I was mostly marking them as ` # L` logical steps which aren't detected by the script.

Reply:
Aren't runtime/build deps already delineated by '=> :build'? 

Not too fussed either way, just wondering about the exact implications of '# R' so we can be on the same page here.

Reply:
Yes but before the script was used runtime deps were added manually, and because builds weren't done in clean containers for every build (amazing that was only a couple of years ago), we had no idea of knowing if the deps list was complete. So adding the # R or # L (separately from build only deps ) is a way to know that the dep list has been recently vetted.

Reply:
Fair enough.